configmap/init/sql/czar/privileges.tpl.sql

```diff
@@ -10,7 +10,7 @@ GRANT ALL ON qservMeta.* TO 'qsmaster'@'localhost';
 GRANT ALL ON qservCssData.* TO 'qsmaster'@'localhost';
 
 -- Create user for external monitoring applications
-CREATE USER IF NOT EXISTS 'monitor'@'localhost' IDENTIFIED BY '<MYSQL_MONITOR_PASSWORD>';
+CREATE USER 'monitor'@'localhost' IDENTIFIED BY '<MYSQL_MONITOR_PASSWORD>';
```

Comment:
Same comment as before regarding `IF NOT EXISTS`. My understanding is that this option is available in MySQL as of version `5.7.6` and in MariaDB since version `10.1.3`.

Reply:
Same thing, this script should only be triggered against an empty database. So I prefer it crashes if data exists because this is not the desired behaviour.